// @spotify-internal/encore-web/es/components/FormInput/Input.js
import {
    screenSmMin,
    screenXsMax,
    spacer4,
    spacer8,
    spacer12,
    spacer48
} from '@spotify-internal/encore-foundation';
import styled from 'styled-components';
import {
    formControlBase,
    formControlTheme,
    rootStyle
} from "../../styles";
var formInputPadding = 14;
export var Input = styled.input.withConfig({
    displayName: "Input",
    componentId: "sc-1gbx9xe-0"
})(["", ";", ";margin-block-start:0;margin-block-end:0;border-radius:", ";padding:", "px;", ";&[type='date'],&[type='month'],&[type='time'],&[type='week']{@media (min-width:", "){padding-block-end:", ";}}&[type='color']{@media (max-width:", "){padding:0;border-block-end:0;&:focus,&:hover:focus{padding:0;}@media not all and (min-resolution:0.001dpcm){@supports (-webkit-appearance:none){border-block-end:1px solid;padding-block-start:10px;padding-block-end:9px;&:focus,&:hover:focus{padding-block-start:10px;padding-block-end:", ";border-block-end-width:2px;}}}}@media (min-width:", "){padding:", ";block-size:", ";inline-size:", ";@media not all and (min-resolution:0.001dpcm){@supports (-webkit-appearance:none){inline-size:100%;padding:14px;}}}}&[type='file']{padding-inline-start:0;padding-inline-end:0;padding-block-start:", ";padding-block-end:0;line-height:17px;@media (max-width:", "){border-block-end:0;}@media (min-width:", "){box-shadow:none;}&:focus,&:hover:focus{@media (min-width:", "){box-shadow:none;}}}"], formControlBase, rootStyle(), spacer4, formInputPadding, function(props) {
    return formControlTheme({
        isUsingKeyboard: props.isUsingKeyboard,
        isHover: props.isHover,
        isFocus: props.isFocus,
        error: props.error
    });
}, screenSmMin, spacer12, screenXsMax, spacer8, screenSmMin, spacer4, spacer48, spacer48, spacer8, screenXsMax, screenSmMin, screenSmMin);
